tests/phpunit/locallib_test.php

```diff
@@ -209,8 +213,8 @@ public function check_whitelisted_ip_redirect_testcases() {
             'saml off, no ip, no redirect'              => ['off', '', false],
             'saml off, getremoteaddr(), no redirect'    => ['off', getremoteaddr(), false],
             'saml not specified, junk, no redirect'     => [null, 'qwer1234!@#$qwer', false],
-            'saml not specified, junk+ip, yes redirect' => [null, 'qwer1234!@#$qwer'."\n".getremoteaddr(), true],
-            'saml not specified, localip, yes redirect' => [null, getremoteaddr()."\n127.0.0.1\n0.0.0.0", true],
+            'saml not specified, junk+ip, yes redirect' => [null, 'qwer1234!@#$qwer,127.0.0.1', true],
+            'saml not specified, localip, yes redirect' => [null, '127.0.0.,1.2.3.4', true],
```

Comment:
see above, pretty sure this should be a \n not a comma

Reply:
By moving to using ip_utils::is_ip_in_subnet_list, updated tests to reflect new lines